src/frontend/components/Header/Header.css

```diff
@@ -1,80 +1,62 @@
 
 .Header {
-    height: 15vh;
-    padding: auto 20%;
-  }
-  
-  .navbar {
-    display: flex;
-    justify-content: space-between;
-    padding: 10px;
-  }
-  
-  .navbar-join {
-    display: flex;
-    color: #ffffff;
-    text-align: right;
-    width: 300px;
-    justify-content: space-evenly;
-  }
+  height: 15vh;
+  padding: 5px;
+}
+  
+.navbar {
+  height: 15vh;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+}
 
+.navbar-join {
+  display: flex;
+  color: #ffffff;
+  text-align: right;
+  width: 300px;
+  justify-content: space-evenly;
+}
 
-  
-  .nav-button{
-    text-decoration: none;
-    background-color: #1e1e1e;
-    color: white;
-    border: none;
-    display: inline;
-    font-size: 1.1em;
+.nav-button{
+  text-decoration: none;
+  background-color: #1e1e1e;
+  color: white;
+  border: none;
+  display: inline;
+  font-size: 1.1em;
+}
+
+.logo-img {
+  width: 40vw;
+}
+
+.user-icon {
+  display: none;
+  color: white;
+}
+
+.nav-button {
+  font-size: 1.3em;
+  padding: 0 10px;
+}
+/* Desktop */
+@media (min-width: 600px) {
+  .navbar {
+    padding: auto 0;
+    width: 100%;
+    margin: 0;
   }
-  
- 
-  
+
   .logo-img {
-    height: auto;
-  }
-  
-  .user-icon {
-    display: none;
-    color: white;
-  }
-  
-  .Search-input {
-    border: white;
-    width: 80%;
-    height: 30px;
-    margin-top: 10px;
-  }
-  
-  .searchbtn {
-    width: 80%;
-    background: white;
-    
+    width: 200px;
   }
-  
-  @media (min-width: 600px) {
-
-    .navbar {
-      width: 80%;
-      margin: 0 auto;
-    }
-
-    .logo-img {
-      height: 50px;
-    }
 
-    .signing {
-      font-size: 20px;
-      text-align: right;
-      padding: 30px;
-    }
-    
-    .Search-input {
-      width: 300px;
-      height: 50px;
-      margin-top: 10px;
-    }
-  
+  .signing {
```

Comment:
I thought that we changed this class name in `sign-in` as per the convention.
Is this a remain that was not changed or this class is out of the convention for naming?